Client.php

```diff
@@ -102,22 +101,20 @@ public function __construct ( Socket\Client $client, $endPoint = '/',
     /**
      * Connect, i.e. open the connection and do handshake.
      *
-     * @access  public
      * @return  void
      */
-    public function connect ( ) {
```

Comment:
By the way one thing I noticed is PHP_CodeSniffer was not removing the space as it fixed this. So the fixed version was like `public function connect( )` .